### DISCUSSION TITLE
part-ii/2013-29/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#318] [Question comments]
Part II 2013 Coding and Cryptography: Paper 2, Section II, H 

https://questions.tripos.org/part-ii/2013-29/

**Comment by DannyDuberstein** [comment]
Seems really unfair if you haven't done Galois theory?

**Reply by DannyDuberstein** [reply]
Idea is 31st cyclotomic factors into irreducible quintics mod 2. By Frobenius, roots of each factor come in cosets of 2 in multiplicative group. Checking the cosets that 1,2,...,7 are in, we find we have 8,9,10 too, so this is the same code as BCH of design distance 10.

**Reply by DannyDuberstein** [reply]
*11
